ultralytics/nn/modules/head.py

```diff
@@ -470,7 +470,7 @@ def _generate_anchors(self, shapes, grid_size=0.05, dtype=torch.float32, device=
 
             valid_WH = torch.tensor([w, h], dtype=dtype, device=device)
             grid_xy = (grid_xy.unsqueeze(0) + 0.5) / valid_WH  # (1, h, w, 2)
-            wh = torch.ones_like(grid_xy, dtype=dtype, device=device) * grid_size * (2.0**i)
+            wh = torch.ones_like(grid_xy, dtype=dtype, device=device) * grid_size * (1 << i)
```

Comment:
is this faster than `2.0**i`? asking because `1 << i` is less readable, but we might want it if it's faster.